syntax = "proto3";

service WordSplitter {
    rpc GetSplitted(OriginString) returns (KeyWords) {}
    rpc SayHello(HelloRequest) returns (HelloReply) {}
}

message HelloReply
{
    string message =1;

}

message HelloRequest
{
    string name =1;
}

message OriginString {
    string origin= 1;
}

message KeyWords{
    repeated string keyWords=1;
}